tests/js/jest.config.js

```diff
@@ -44,7 +44,7 @@ export default {
   // See https://jestjs.io/docs/configuration#transformignorepatterns-arraystring
   transformIgnorePatterns: ['/node_modules/(?!(use-reduction)/)'],
   testEnvironment: 'jsdom',
-  testMatch: ['**/test/**/*.[jt]s'],
+  testMatch: ['**/test/**/*.[jt]s?(x)'],
```

Comment:
Due to enabling .tsx files, the following tests are now failing:
https://github.com/GoogleForCreators/web-stories-wp/actions/runs/3824643543/jobs/6506964376#step:9:286

Thoughts @barklund @swissspidy?

Since it's not related to this PR, I'll just revert it back, however, these need to get fixes.

Created an issue for it: #12895 12895